p2p/net/upgrader/upgrader.go

```diff
@@ -174,6 +175,7 @@ func (u *upgrader) upgrade(ctx context.Context, t transport.Transport, maconn ma
 	smconn, err := u.setupMuxer(ctx, sconn, server, connScope.PeerScope())
 	if err != nil {
 		sconn.Close()
+		canonicallog.LogMisbehavingPeerNetAddr(p, conn.RemoteAddr(), "muxer", err, "failed to setup muxer")
```

Comment:
Should we remove this one, for the same reason we moved the log to the security transport? Setting up the muxer is not really expensive, so there's no real attack vector here anyway.